.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
+vendor
+composer.lock
+phpunit.xml
```

Comment:
CS: trailing newline is missing (and in quite some other files as well)

Reply:
Ego sincere me, I'll look into it (thought that rule only applied to PHP files)